apps/backoffice-v2/src/pages/Root/ServerDown.layout.tsx

```diff
@@ -4,9 +4,9 @@ import { Outlet } from 'react-router-dom';
 import { ErrorAlert } from '../../common/components/atoms/ErrorAlert/ErrorAlert';
 
 export const ServerDownLayout: FunctionComponent = () => {
-  const { isSuccess } = useHealthQuery();
+  const { isSuccess, isLoading } = useHealthQuery();
 
-  if (isSuccess) return <Outlet />;
+  if (isSuccess || isLoading) return <Outlet />;
```

Comment:
Are you sure we want to show `<Outlet />` when still loading?
Maybe a loader or something like that instead?
What happens on periodic check? Will the user experience a flicker?